Tests/biddingTests.js

```diff
@@ -0,0 +1,43 @@
+var homePage = require('../Pages/homePage.js'),
+    loginPage = require('../Pages/loginPage'),
```

Comment:
Errors in console:
<img width="843" alt="Screenshot 2020-12-06 at 18 31 28" src="https://user-images.githubusercontent.com/50828644/101287669-f90da880-37f1-11eb-894b-c0336aa81bec.png">

Reply:
I added stability to the tests